--- xmlto-0.0.22/xmlto.in.orig	2009-03-24 15:54:31.000000000 +0100
+++ xmlto-0.0.22/xmlto.in	2009-10-29 01:09:19.000000000 +0100
@@ -124,7 +124,7 @@
 XSLTPROC_PATH=@XSLTPROC@
 
 # Try to setup papersize using libpaper first ...
-if [ -n "`which "$PAPERCONF_PATH" 2>/dev/null`" ]
+if [ -n "$PAPERCONF_PATH" ]
 then
   papername=`"$PAPERCONF_PATH" -n`
   paperheight=`"$PAPERCONF_PATH" -mh | sed 's/ //g'`
@@ -155,7 +155,7 @@
   fi
 
 # ... or use magic paper size, based on LC_PAPER
-elif [ -n "`which "$LOCALE_PATH" 2>/dev/null`" ]
+elif [ -n "$LOCALE_PATH" ]
 then
   # For paper sizes we know about, specify them.
   h=$("$LOCALE_PATH" LC_PAPER 2>/dev/null | head -n 1)
@@ -178,7 +178,7 @@
 fi
 
 # Magic encoding, based on locale
-if [ -n "`which "$LOCALE_PATH" 2>/dev/null`" ]
+if [ -n "$LOCALE_PATH" ]
 then
   charmap=$("$LOCALE_PATH" charmap 2>/dev/null)
 
@@ -227,14 +227,14 @@
 PDFXMLTEX_PATH=@PDFXMLTEX@
 
 #check if we could use fop/dblatex backend as default(if not, use passivetex)
-if [ x"$USE_BACKEND" = xFOP ] && [ -z "`which "$FOP_PATH" 2>/dev/null`" ]
+if [ x"$USE_BACKEND" = xFOP ] && [ -z "$FOP_PATH" ]
 then
   echo >&2 "@PACKAGE@: Warning: fop not found or not executable."
   echo >&2 "@PACKAGE@: Using default backend..."
   USE_BACKEND=DEFAULT
 fi
 if [ x"$USE_BACKEND" = xDBLATEX ] && \
-   [ -z "`which "$DBLATEX_PATH" 2>/dev/null`" ]
+   [ -z "$DBLATEX_PATH" ]
 then
   echo >&2 "@PACKAGE@: Warning: dblatex not found or not executable."
   echo >&2 "@PACKAGE@: Using default backend..."
@@ -328,7 +328,7 @@
 	;;
   --with-fop)
 	##use fop instead of passivetex where possible
-	if [ -z "`which "$FOP_PATH" 2>/dev/null`" ]
+	if [ -z "$FOP_PATH" ]
 	then
 		echo >&2 Warning: fop not found or not executable.
 		echo >&2 Using default backend...
@@ -339,7 +339,7 @@
 	;;
   --with-dblatex)
 	##use dblatex instead of passivetex where possible
-	if [ -z "`which "$DBLATEX_PATH" 2>/dev/null`" ]
+	if [ -z "$DBLATEX_PATH" ]
 	then
 		echo >&2 Warning: dblatex not found or not executable.
 		echo >&2 Using default backend...
@@ -487,7 +487,7 @@
 if [ "$SKIP_VALIDATION" -eq 0 ] && [ "$SOURCE_FORMAT" != "fo" ]
 then
   #do we have xmllint validation tool?
-  if [ -z "`which "$XMLLINT_PATH" 2>/dev/null`" ]
+  if [ -z "$XMLLINT_PATH" ]
   then
     echo >&2 "@PACKAGE@: xmllint validation tool not found or not executable."
     echo >&2 "@PACKAGE@: Skipping validation... " \
@@ -519,7 +519,7 @@
 else
 
   #do we have xsltproc tool?
-  if [ -z "`which "$XSLTPROC_PATH" 2>/dev/null`" ]
+  if [ -z "$XSLTPROC_PATH" ]
   then
     echo >&2 "@PACKAGE@: Can't continue, xsltproc tool not found or not executable."
     exit 3
